### PR TITLE
Feature/jp support cucumber nested steps

### PR DESCRIPTION
**Pull request by jpaolini**
## Proposed changes
ReportPortal release v5 comes with the ability to have nested steps which for Cucumber is useful so that the reporting is at the Scenario level rather than the Step level.  Currently the reporting displays the total number of Steps for all Scenarios and this allows it to display the total number of Scenarios instead.  It follows the approach used in the [reportportal-agent-cucumber](https://github.com/reportportal/agent-js-cucumber).  A new flag, `cucumberNestedSteps`, was added since the implementation is specific to cucumber.

Another addition is an option to add an attribute with the feature name to each scenario so that that Component Health widget can be used to display the health of each Feature.  To use this feature, it is required to have the nested steps option (`cucumberNestedSteps`) also applied.

### CODE REVIEW

**Comment by jpaolini** [comment]
Fixing the linting now

**Comment by BorisOsipov** [comment]
Hi @jpaolini 

Thanks for these features! I'll review PR at the weekends

File: lib/ReporterOptions.ts [hunk 1]
```diff
@@ -8,5 +8,7 @@ export default class ReporterOptions {
   public seleniumCommandsLogLevel = LEVEL.DEBUG;
   public parseTagsFromTestTitle = false;
   public setRetryTrue = false;
+  public cucumberNestedSteps = false;
```
Comment (BorisOsipov) [inline lib/ReporterOptions.ts]: could you add these in readme docs too?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wdio-reportportal-reporter",
-  "version": "5.2.4",
+  "version": "5.3.0",
```
Comment (BorisOsipov) [inline package.json]: don't change. it will be bumped automatically in release script :)

**Comment by BorisOsipov** [comment]
@jpaolini thank you!

Will release this soon

**Pull request merged**